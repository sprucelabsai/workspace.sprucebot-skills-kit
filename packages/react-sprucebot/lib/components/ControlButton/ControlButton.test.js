'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _reactTestRenderer = require('react-test-renderer');

var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

require('jest-styled-components');

var _ControlButton = require('./ControlButton');

var _ControlButton2 = _interopRequireDefault(_ControlButton);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

test('it renders', function () {
	var tree = _reactTestRenderer2.default.create(_react2.default.createElement(_ControlButton2.default, null)).toJSON();
	expect(tree).toMatchSnapshot();
});

test('it renders a link', function () {
	var tree = _reactTestRenderer2.default.create(_react2.default.createElement(_ControlButton2.default, { href: 'https://sprucebot.com' })).toJSON();
	expect(tree).toMatchSnapshot();
});