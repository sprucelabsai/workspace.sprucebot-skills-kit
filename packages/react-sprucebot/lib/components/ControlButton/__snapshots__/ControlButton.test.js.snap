// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: none;
  color: #00aac7;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  font-weight: normal;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  position: relative;
  margin: 0;
  padding: 0;
  width: auto;
}

.c0:hover {
  background: none;
}

<button
  className="ControlButton control-button  c0"
  onClick={undefined}
  tabIndex={0}
>
  <span />
</button>
`;

exports[`it renders a link 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #00aac7;
  cursor: pointer;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  position: relative;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<a
  className="ControlButton control-button  c0"
  href="https://sprucebot.com"
  onClick={undefined}
  tabIndex={0}
>
  <span />
</a>
`;
